### PR TITLE
Fix typo in docs for wasm proxy configuration

### PR DESCRIPTION
**Pull request by anuraaga**
Signed-off-by: Anuraag Agrawal <anuraaga@gmail.com>

The reference doesn't match the API spec

https://github.com/proxy-wasm/spec/blob/56ef3547de853d4d989a5b866f510d3ae713d8d4/abi-versions/vNEXT/README.md#proxy_on_configure

### CODE REVIEW

**Review by daixiang0 — APPROVED** [review]
lgtm

File: api/envoy/extensions/wasm/v3/wasm.proto [hunk 1]
```diff
@@ -137,7 +137,7 @@ message PluginConfig {
   }
 
   // Filter/service configuration used to configure or reconfigure a plugin
-  // (proxy_on_configuration).
+  // (proxy_on_configure).
```
Comment (phlax) [inline api/envoy/extensions/wasm/v3/wasm.proto:140]:
```suggestion
  // (``proxy_on_configure``).
```

**Review by phlax — DISMISSED** [review]
lgtm, thanks @anuraaga

**Comment by phlax** [comment]
@anuraaga looks like there is a branch conflict

**Comment by phlax** [comment]
@markdroth would you mind stamping this one

**Review by phlax — APPROVED** [review]
lgtm, thanks @anuraaga

**Pull request merged**